src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        int massiveSize = 6;
```

Comment:
All magic numbers in your code should be constants. It's better to define '6' as a named constant to give it a meaningful context and to avoid magic numbers.